.github/workflows/instrumented-tests.yml

```diff
@@ -0,0 +1,142 @@
+# Workflow for running Android instrumented tests across multiple API levels
```

Comment:
instrumented tests will run on:
- merge/push to master
- merge/push to rel/*
- any PR labelled `test:all` (only once, so to run it again you gotta remove it and re-add. Did this on purpose bc otherwise this workflow will show up on ALL PR's as "skipped" which is kinda annoying)